doc_source/create-cluster.md

```diff
@@ -196,32 +200,35 @@ You might receive an error that one of the Availability Zones in your request do
 
    The `keyArn` member can contain either the alias or ARN of your CMK\. The CMK must be symmetric, created in the same Region as the cluster, and if the CMK was created in a different account, the user must have access to the CMK\. For more information, see [Allowing users in other accounts to use a CMK](https://docs.aws.amazon.com/kms/latest/developerguide/key-policy-modifying-external-accounts.html) in the *AWS Key Management Service Developer Guide*\. Kubernetes secrets encryption with an AWS KMS CMK requires Kubernetes version 1\.13 or later\.
 
+**Important**  
+Deletion of the customer master key \(CMK\) will permanently put the cluster in a degraded state. If any customer master keys used for cluster creation are scheduled for deletion, please double check and verify that this is the intended action. Once the key is deleted, there is no path to recovery for the cluster. 
```

Comment:
Shouldn't this be customer managed key?

Reply:
It is correct as per https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html